src/PageHeader/PageHeader.stories.tsx

```diff
@@ -0,0 +1,413 @@
+    },
+    ParentLink: {
+      type: 'string',
+      if: {arg: 'ContextArea.hidden', truthy: false},
+      table: {
+        category: 'ContextArea Slot'
+      },
+      description: 'The default way to let users navigate up in the hierarchy on Narrow viewports.'
+    },
+    'ParentLink.hidden': {
+      type: 'boolean',
+      if: {arg: 'ContextArea.hidden', truthy: false},
+      table: {
+        category: 'ContextArea Slot',
+        defaultValue: {
+          summary: `
+        {
+          narrow: false,
+          regular: true,
+          wide: true
+        }
+        `
+        }
+      },
+      description: 'Parent '
+    },
+    'ContextBar.hidden': {
+      type: 'boolean',
+      if: {arg: 'ContextArea.hidden', truthy: false},
+      table: {
+        category: 'ContextArea Slot',
+        defaultValue: {
+          summary: `
+        {
+          narrow: false,
+          regular: true,
+          wide: true
+        }
+        `
+        }
+      },
+      description:
+        'ContextBar is generic slot for any component above the title region. Use it for custom breadcrumbs and other navigation elements instead of ParentLink.'
+    },
+    'ContextAreaAction.hidden': {
+      type: 'boolean',
+      if: {arg: 'ContextArea.hidden', truthy: false},
+      table: {
+        category: 'ContextArea Slot',
+        defaultValue: {
+          summary: `
+        {
+          narrow: false,
+          regular: true,
+          wide: true
+        }
+        `
+        }
+      }
+    },
+    'BackButton.hidden': {
+      type: 'boolean',
+      table: {
+        category: 'TitleArea Slot',
+        defaultValue: {
+          summary: `
+        {
+          narrow: true,
+          regular: false,
+          wide: false
+        }
+        `
+        }
+      },
+      description:
+        'A back button can be used as a leading action for local navigation. On Narrow viewports, use parentLink instead.'
+    },
+    Title: {
+      type: 'string',
+      table: {
+        category: 'TitleArea Slot'
+      }
+    },
+    'Title.as': {
+      control: {
+        type: 'select'
+      },
+      options: ['h1', 'h2', 'h3', 'h4', 'h5', 'h6'],
+      table: {
+        category: 'TitleArea Slot'
+      }
+    },
+    'Title.variant': {
+      control: {
+        type: 'radio'
+      },
+      options: ['large', 'medium', 'subtitle'],
+      table: {
+        category: 'TitleArea Slot',
+        defaultValue: {
+          summary: `
+          {
+            narrow: 'medium',
+            regular: 'large',
+            wide: 'large'
+          } `
+        }
+      },
+      description:
+        '`medium` is the most common page title size. Use for static titles in most situations. `large` for for user-generated content such as issues, pull requests, or discussions. `subtitle` when a PageHeader.Title is already present in the page, such as in a SplitPageLayout.'
+    },
+    LeadingVisual: {
```

Comment:
@vdepizzol I wonder if a dropdown control with a few choice of Octicons provides a good experience here? Because LeadingVisual can be Avatars or other custom visuals as you mentioned on the Figma specs. Same goes for `TraliningVisual` . I added a description attribute to give some more info (see a few lines below.) but I am not sure if that would be helpful. Curious to hear your thought on that 👀

Reply:
I think the Octicons work well and cover a majority of the use cases for a PageHeader's LeadingVisual